### ISSUE TITLE
User management view: Rethink quota

### ISSUE THREAD

**Issue by kesselb** [nextcloud/server#18540]
**Is your feature request related to a problem? Please describe.**

![Screenshot from 2019-12-22 19-56-22](https://user-images.githubusercontent.com/3902676/71326139-fb9ac380-24f6-11ea-809d-cea2a3b7d4bb.png)

I like the new user management view :+1: but I'm unhappy with the quota bar. Please switch from a designer to a user manager perspective for a moment. I'm not able to see what quota is assigned for a user or how many storage is used. 

- To see the storage in use (to concrete value): hover over the bar.
- To see the assigned quota: enter the edit mode.

**Describe the solution you'd like**
Remove the progress bar. Show the important information: `x MB of y MB used`. The progress bar is fine for the user view but not user management view.

**Describe alternatives you've considered**
There is no alternative to consider.

cc @nextcloud/designers

**Comment by skjnldsv** [comment]
@jancborchardt

**Comment by jancborchardt** [comment]
What was wrong with the way we had it before, text and visualization combined?
![quota](https://user-images.githubusercontent.com/925062/71576982-fc4dde00-2b24-11ea-81a7-6f70f0a027ca.png)

We should have it like this again and we get the best of both worlds. The correct information, and a quick identification of who is close to being over quota.

And yes, the text could be:
- 11 GB used of unlimited
- 2 GB used of 5 GB

**Comment by GretaD** [comment]
I remember discussing to show the used space as tooltip but not removing the general space. So maybe we removed it accidentally.
So @jancborchardt  @kesselb, you propose to show both (general space and used space) inside the field and not as a tooltip?

**Comment by kesselb** [comment]
> I Remove the progress bar. Show the important information: x MB of y MB used. The progress bar is fine for the user view but not user management view.

That's my proposal. I don't see any need for a quota bar at this place. Probably from a designer perspective this quota bar makes sense. But it's hiding important information.

**Comment by jancborchardt** [comment]
Yes, my proposal is as per my last comment in https://github.com/nextcloud/server/issues/18540#issuecomment-569630626 :)

And that does include the bar, which is not just decoration but great for quick identification on who is close to hitting limits – and as we had before.

**Comment by kesselb** [comment]
For the user management view it's decoration. To see if someone is close to hitting the limits is not the point. The point is how many quota is assigned and how much used. Everyone can do the math ;) I'm fine with using a decent color to indicate if someone is hitting the quota (e.g yellow < 90%, red < 95%).

I know you have a lot of expertise in this area. My concern is to not hide important information behind some visual elements. Making it harder than necessary to get information.
